sdcm/cluster.py

```diff
@@ -2325,6 +2326,16 @@ def create_swap_file(self, size=1024):
         if "sct_configured_swapfile" not in result.stdout:
             self.log.warning("Swap file is not used on loader node %s.\nError details: %s", self, result.stderr)
 
+    def change_hostname(self):
```

Comment:
If you put it here it will run for any class that inherits from BaseNode. 
Ask yourself: do we need this method to run in GCE? Do we need this in Docker? The answer is no.
So better:
- make this method abstract (raise not implemented error) in BaseNode
- change the name to `set_hostname` (we *set*/configure the name of the host)
- for AWS use this implementation
- for GCE skip

Reply:
Done